features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -129,6 +129,11 @@
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.folding/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-foobot" description="Foobot Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
```

Comment:
@kaikreuzer should this binding depend on `<feature>openhab-transport-http</feature>` or is that for real specific things?